src/arkade-lightning.ts

```diff
@@ -1023,4 +1089,26 @@ export class ArkadeLightning {
                 });
         }
     }
+
+    /**
+     * Validate we are using a x-only public key
+     * @param publicKey
+     * @param keyName
+     * @param swapId
+     * @returns Uint8Array
+     */
+    private normalizeToXOnlyPublicKey(
+        publicKey: Uint8Array,
+        keyName: string,
+        swapId?: string
+    ): Uint8Array {
+        if (publicKey.length === 33) {
+            return publicKey.slice(1);
+        } else if (publicKey.length !== 32) {
```

Comment:
```suggestion
        } else if (publicKey.length !== 32) {
```

```suggestion
        } 

       if (publicKey.length !== 32) {
```
nit: else not needed